### ISSUE TITLE
Figure issue about your paper (Figure 4 and Figure 15)

### ISSUE THREAD

**Issue by MatthewYZhang** [huggingface/datablations#7]
Hi,

I am reading your paper and I have noticed that figure 4 and figure 15 are exactly the same. Are they meant to be the same? I believe that figure 4 shows the result of models trained on a different training set rather than OSCAR corpus (because you mentioned in Appendix I that 'To ensure our findings are not dataset-dependent, we train models with the same configurations from Figure 4 on the OSCAR corpus'). I wonder if you placed a wrong figure here. 

Appreciate your work! I believe this work will definitely give researchers and engineers more insights when training and developing LLMs. 

Matthew

**Comment by Muennighoff** [comment]
You're right, thanks a lot for noting! Attached is the correct OSCAR graph. I will update the paper soon.


[validation_oscar.pdf](https://github.com/huggingface/datablations/files/11645255/validation_oscar.pdf)

Edit: The paper has been updated :)
